christulin/tic-tac-toe
# Placed an open nav at the top to trigger states, while in dev

# General Purpose of Pull Request:
Top nav to make it easier for us to toggle views. 

# Proposed Changes
Development QoL 

# How to test?
Click on the nav items to make sure each view triggers

![image](https://user-images.githubusercontent.com/14282966/103035161-928bc880-451b-11eb-8eee-e8676d46ce3c.png)
![image](https://user-images.githubusercontent.com/14282966/103035166-9586b900-451b-11eb-86c1-9bb34232db7d.png)
